easybuild/easyconfigs/l/libdwarf/libdwarf-20140805-GCC-4.9.2.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'libdwarf'
+version = '20140805'
+
+homepage = 'http://www.prevanders.net/dwarf.html'
+description = """The DWARF Debugging Information Format is of interest to programmers working on compilers
+and debuggers (and anyone interested in reading or writing DWARF information))"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.prevanders.net']
+
+dependencies = [('libelf', '0.8.13')]
+
+with_configure = True
+preconfigopts = 'CFLAGS="-fPIC $CFLAGS" '
+configopts = "--enable-shared "
+
+# This is dirty but libdwarf cannot find it's own library in the build process...
+prebuildopts = ' LD_LIBRARY_PATH="../libdwarf:$LD_LIBRARY_PATH" '
```

Comment:
Yeah, well, at least you're making clear why it's needed, and I've been worse. ;-)